debezium-core/src/main/java/io/debezium/pipeline/metrics/ChangeEventSourceMetricsMXBean.java

```diff
@@ -15,7 +15,7 @@
     String getLastEvent();
     long getMilliSecondsSinceLastEvent();
     long getTotalNumberOfEventsSeen();
-    long getNumberOfEventsSkipped();
```

Comment:
I think this is misunderstanding here. The `getNumberOfEventsSkipped()` should stay but the implementation should return `numberOfEventsFiltered.get()` - this is to keep 0.9.x backward compatible. DBZ-1209 shuld be implemented in 0.10 as the breaking change.

Reply:
I added it back to the interface marked as deprecated along with re-implementing it in Metrics.java as a pass through to getNumberOfEventsFiltered()